fritz.defaults.yaml

```diff
@@ -95,6 +95,8 @@ skyportal:
     #
     # You need to have Google+ API enabled; it takes a few minutes to activate.
 
+    host: private.caltech.edu
```

Comment:
do we want to have this in version control?

Reply:
I believe nothing criminal, but took it out anyways, thanks.